community/bolt/src/main/java/org/neo4j/bolt/v1/runtime/concurrent/RunnableBoltWorker.java

```diff
@@ -94,12 +96,21 @@ public void run()
                 }
             }
         }
-        catch ( Throwable e )
+        catch ( BoltConnectionAuthFatality e )
+        {
+            // this is logged in the SecurityLog
+        }
+        catch ( BoltProtocolBreachFatality e )
+        {
+            log.error( "Bolt protocol breach in session '" + machine.key() + "'" );
+        }
+        catch ( Throwable t )
         {
-            log.error( "Worker for session '" + machine.key() + "' crashed: " + e.getMessage(), e );
```

Comment:
This message seems to have been dropped from the Throwable case. Was that intentional?